.github/workflows/push.yml

```diff
@@ -1,23 +1,30 @@
 ---
-name: Build
-on: [push]
+name: Style
+on: [push,  pull_request]
 
 jobs:
   linter:
     runs-on: ubuntu-latest
-    strategy:
-      matrix:
-        python: ['3.6', '3.7']
     steps:
       - uses: actions/checkout@v1
-      - name: Use Python ${{ matrix.python }}
+
+      - name: Use Python 3.6
```

Comment:
Back to single version ? I thought you said there was a policy ?

Reply:
There is a python projects template (https://github.com/linz/template-python-hello-world) to inform python project set-up. The use of the matrix was as-per this. Initial reading of QGIS packaging indicated that QGIS3 is using 3.6. It was at this stage I removed the matrix as QGIS3.7 did not seem all that useful for this reason. 

However reading this morning it is clear that QIGS3 will run with any version of py3 and some users may alter there install to user other version (not 3.6) of python. Therefore perhaps there is use for a matrix but this is only linting in this job. The functional tests will be against the dockerised QGIS application.

Reply:
> > Beside the `import Config` changed back to `from . import Config`, which breaks running `LINZ/gazetteer/gui/DatabaseConfiguration.py` as done by `.docker/qgis/scripts/docker-entry.sh`,
> 
> I have been doing my testing on a local install of QGIS3 rather than the docker image. `import Config` throws the 'No module named 'Config'' and `from . import Config` fixes this.
> This is inline with my understanding of py3 relative imports where a single dot means that the module or package referenced is in the same directory as the current location

Is  `current location` the location of the script or the CWD of the process ?
Could you please try to make it work in `docker-compose up` ? As it fails there... What's different between the docker container and your local setup ?

Reply:
> Is current location the location of the script or the CWD of the process ?
Could you please try to make it work in docker-compose up ? As it fails there... What's different between the docker container and your local setup ?

I had a few issues today getting docker-compose to build the containers on my machine. I will work on this tomorrow